keyboards/model_f_labs/f62/readme.md

```diff
@@ -0,0 +1,16 @@
+# model_f_labs/f62
+
+![model_f_labs/f62](https://images2.imgbox.com/e5/97/1hPUzXct_o.jpg)
+
+A short description of the keyboard/project
+
+* Keyboard Maintainer: [Purdea Andrei](https://github.com/purdeaandrei)
+* Build Maintainer: Matthew J Wolf
+* Hardware Supported: Supports the brand new Model F reproductions from https://www.modelfkeyboards.com/, using the smaller xwhatsit controller developed by wcass.
+* Hardware Availability: https://www.modelfkeyboards.com/
+
+Make example for this keyboard (after setting up your build environment):
+
+    make model_f_labs/f62:default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware?  This is needed, as a new user may not be aware of how to enter to bootloader to flash new firmware.

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```

Reply:
Bootloader details added to readme.md.